datasets/sent_comp/sent_comp.py

```diff
@@ -0,0 +1,155 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google Sentence Compression dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import gzip
+import json
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{filippova-altun-2013-overcoming,
+    title = "Overcoming the Lack of Parallel Data in Sentence Compression",
+    author = "Filippova, Katja  and
+      Altun, Yasemin",
+    booktitle = "Proceedings of the 2013 Conference on Empirical Methods in Natural Language Processing",
+    month = oct,
+    year = "2013",
+    address = "Seattle, Washington, USA",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/D13-1155",
+    pages = "1481--1491",
+}
+"""
+
+_DESCRIPTION = """\
+Large corpus of uncompressed and compressed sentences from news articles.
+"""
+
+_HOMEPAGE = "https://github.com/google-research-datasets/sentence-compression"
+
+
+_URLs = {
+    datasets.Split.VALIDATION: [
+        "https://github.com/google-research-datasets/sentence-compression/raw/master/data/comp-data.eval.json.gz"
+    ],
+    datasets.Split.TRAIN: [
+        f"https://github.com/google-research-datasets/sentence-compression/raw/master/data/sent-comp.train{str(i).zfill(2)}.json.gz"
+        for i in range(1, 11)
+    ],
+}
+
+
+class SentenceCompression(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class SentComp(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `sent_comp`